### PR TITLE
chore(colorMap): add unit tests

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
image

### CODE REVIEW

File: src/utils/colors-map.spec.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+    expect(colorByDepth).toEqual(colorForTheSameTile);
+  });
+
+  test("Should return DEFAULT color in _getColorByTile method if colorMap isEmpty", () => {
+    const colorMap = new ColorMap();
+
+    const tile = {
+      id: 'test'
+    };
```
Comment (belom88) [inline src/utils/colors-map.spec.ts:74]: You could initialize `colorMap` and `tile` in `beforeEach` hook

File: src/utils/colors-map.spec.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  test("Should return selection color in _getCustomColor", () => {
+    const colorMap = new ColorMap();
+
+    const tile = {
+      id: 'test',
+    };
+
+    const customColor = colorMap.getColor(tile, { coloredBy: 3, selectedTileId: 'test' });
+    const map = colorMap.colorMap
+    const isCustomColorExistInColorMap = Boolean(map['test']);
```
Comment (belom88) [inline src/utils/colors-map.spec.ts:140]:
```suggestion
    const isCustomColorExistInColorMap = Boolean(colorMap.colorMap['test']);
```

**Review by belom88 — APPROVED** [review]

File: src/utils/colors-map.spec.ts (hunk 1 above)
Comment (dryabinin-actionengine) [reply src/utils/colors-map.spec.ts]: Done

File: src/utils/colors-map.spec.ts (hunk 2 above)
Comment (dryabinin-actionengine) [reply src/utils/colors-map.spec.ts]: Done

**Pull request merged**